src/ok.rs

```diff
@@ -123,3 +161,25 @@ pub fn cmd(gen: Option<SevGeneration>, quiet: bool) -> Result<()> {
         ))
     }
 }
+
+fn check_test(res: &TestResult, quiet: bool) -> bool {
```

Comment:
I think this function should be renamed based on what it does.

Perhaps `emit` or something. The caller is who does the actual checking here.